Disable gdbm support since its never actually used anyways #113824 by
Natanael Copa.

diff -ur tdb-1.0.6.orig/Makefile.am tdb-1.0.6/Makefile.am
--- tdb-1.0.6.orig/Makefile.am	2001-12-11 06:05:13.000000000 +0200
+++ tdb-1.0.6/Makefile.am	2009-07-06 21:10:56.000000000 +0300
@@ -13,7 +13,8 @@
 TDBTORTURE =
 endif
 
-noinst_PROGRAMS=tdbspeed tdbiterate $(TDBTEST) $(TDBTORTURE)
+# tdbspeed $(TDBTEST)
+noinst_PROGRAMS=tdbiterate $(TDBTORTURE)
 bin_PROGRAMS=tdbtool tdbdump
 
 lib_LTLIBRARIES=libtdb.la
